makeflow/src/dag_file.h

```diff
@@ -26,6 +26,12 @@ typedef enum {
 	DAG_FILE_STATE_UP
 } dag_file_state_t;
 
+/* the type of a dependency specified in the mountfile */
+typedef enum {
+	SOURCE_LOCAL,
+	SOURCE_HTTP,
+	SOURCE_UNSUPPORTED
+} source_type;
```

Comment:
Accept. I will fix these naming style problem when you are done with commenting.